pack/tdc/tdcc.json

```diff
@@ -301,6 +301,178 @@
         "text": "Easy / Standard\n[skull]: -X. X is the current Disturbance (to a maximum of 6).\n[cultist]: -4. If you fail, draw the top card of the encounter deck. If you fail by 3 or more, that card gains peril and surge.\n[tablet]: -4. For each point you fail by, you must either take 1 horror or discard 1 card from your hand.\n[elder_thing]: -3. Reveal another token.",
         "type_code": "scenario"
     },
+    {
+        "back_link": "11674b",
+        "code": "11674",
+        "doom": null,
+        "double_sided": true,
+        "encounter_code": "sepulchre_of_the_sleeper",
+        "encounter_position": 2,
+        "faction_code": "mythos",
+        "illustrator": "JB Casacop",
+        "name": "Beneath the City",
+        "pack_code": "tdcc",
+        "position": 174,
+        "quantity": 1,
+        "text": "<b>Forced</b> - when doom would be placed on a card other than a story asset: Place 1 resource on the scenario reference card, as Disturbance, instead. If the current Disturbance is 6 or more (8 or more instead of there are exactly 1 or 2 investigators in the game), immediately advance.\n<b>Objective</b> - Prepare your artifacts for the final showdown!",
```

Comment:
```suggestion
        "text": "<b>Forced</b> - When doom would be placed on a card other than a story asset: Place 1 resource on the scenario reference card, as Disturbance, instead. If the current Disturbance is 6 or more (8 or more instead of there are exactly 1 or 2 investigators in the game), immediately advance.\n<b>Objective</b> - Prepare your artifacts for the final showdown!",
```